### ISSUE TITLE
AvoidTooShortNames rule does not work on short function argument name

### ISSUE THREAD

**Issue by Mersho** [fsprojects/FSharpLint#632]
### Description

The AvoidTooShortNames rule doesn't give warning on a short name function argument.

### Steps To Reproduce

`dotnet fsharplint lint ..\ConsoleApp\ConsoleApp\Program.fs`

ConsoleApp.fsproj
```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net5.0</TargetFramework>
    <WarnOn>3390;$(WarnOn)</WarnOn>
  </PropertyGroup>

  <ItemGroup>
    <Compile Include="Program.fs" />
  </ItemGroup>

</Project>
```

Program.fs
```
open System

type Foo() =
    member this.BarMethod(x: int) =
        x.ToString()
```

### Current Results

```
========== Linting C:\Users\PC\source\repos\ConsoleApp\ConsoleApp\obj\Debug\net5.0\ConsoleApp.AssemblyInfo.fs ==========
========== Finished: 0 warnings ==========
========== Linting C:\Users\PC\source\repos\ConsoleApp\ConsoleApp\Program.fs ==========
========== Finished: 0 warnings ==========
========== Summary: 0 warnings ==========
```

### Expected behavior

The rule should warn about using short identifiers.

### Related information

* Operating system Windows 11
* Branch master
* .NET Runtime, CoreCLR or Mono Version .NET 5
